scripts/phos-compressor-raw-qcmnt3.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# set -x;
+set -e;
+set -u;
+
+WF_NAME=phos-compressor-raw-qcmn-localt3
```

Comment:
I would maybe use this name instead (or even `phos-compressort3-raw-qcmn-local` because the compressor is parallelized).. This way we also leave room for some automatization in GUI, if we always keep the `*-remote` and  `*-local` convention.

This will of course imply generating the templates again.
```suggestion
WF_NAME=phos-compressor-raw-qcmnt3-local
```